README.md

```diff
@@ -118,6 +125,26 @@ end
 
 - `value` - The value to set for the given option. Required for the set action. Booleans are mapped to `on` or `off`.
 
+### gluster\_mountbroker\_user
+
+Use this resource to allow or disallow the given user access to the given volume:
+
+```ruby
+gluster_mountbroker_user 'user/volume_name' do
+  action :add
+end
+```
+
+```ruby
+gluster_mountbroker_user 'user/volume_name' do
+  action :remove
+end
+```
+
+#### Parameters
+
+None.
```

Comment:
That is true of `volume` and `key` in the volume_option resource too but I didn't mention those. In both cases, these properties are determined from the `name` property and I don't believe there is any reason why you would need to state them explicitly so to mention them here would be confusing. To be clear, I did it this way so that users would not need to invent their own unique resource name. Less experienced Chef users may not even realise that the name needs to be unique.